content/docs/en/routing/vue-router.md

```diff
@@ -91,15 +88,28 @@ const router = new VueRouter({
 ```
 
 ---
-And load one of those routes when the app starts:
-```js
+Load one of the routes when the app starts.
+
+```JavaScript
 router.replace('/master');
 ```
 
 ---
-Oh, and don't forget to tell `Vue` about your routes:
-```js
+Tell `Vue` about your routes.
+
+```JavaScript
 new Vue({
   router
 }).$start();
 ```
+
+## See also
+
+Vue Router has more tricks up its sleeve, so be sure to visit [the official documentation](https://router.vuejs.org/en/).
+
+Check out the following [NativeScript-Vue samples](https://github.com/nativescript-vue/nativescript-vue/tree/master/samples):
+
+* [app-with-page-routing](nativescript-vue/samples/app/app-with-page-routing.js)
+* [app-with-router](nativescript-vue/samples/app/app-with-router.js)
+* [app-with-router-pages](nativescript-vue/samples/app/app-with-router-pages.js)
+* [app-with-router-v2](nativescript-vue/samples/app/app-with-router-v2.js)
```

Comment:
Let's not include the v2 yet, because that doesn't actually work - it's just an experiment for the next version

Reply:
Removed.